examples/local/vttablet-up.sh

```diff
@@ -38,23 +38,8 @@ source $script_root/env.sh
 
 init_db_sql_file="$VTROOT/config/init_db.sql"
 
-export EXTRA_MY_CNF=$VTROOT/config/mycnf/default-fast.cnf:$VTROOT/config/mycnf/rbr.cnf
-
-case "$MYSQL_FLAVOR" in
-  "MySQL56")
-    export EXTRA_MY_CNF=$EXTRA_MY_CNF:$VTROOT/config/mycnf/master_mysql56.cnf
-    ;;
-  "MariaDB")
-    export EXTRA_MY_CNF=$EXTRA_MY_CNF:$VTROOT/config/mycnf/master_mariadb.cnf
-    ;;
-  "MariaDB103")
-    export EXTRA_MY_CNF=$EXTRA_MY_CNF:$VTROOT/config/mycnf/master_mariadb103.cnf
-    ;;
-  *)
-    echo "Please set MYSQL_FLAVOR to MySQL56 or MariaDB."
-    exit 1
-    ;;
-esac
+# Previously this file set EXTRA_MY_CNF based on MYSQL_FLAVOR
```

Comment:
I think that `default-fast.cnf` makes some unusual/unsafe recommendations. I am not sure it is easy to followup as PR (as with `master_mysql56.cnf`) because by dropping the environment variable the flavor is not known at this point. I will make it clearer in the description why this is being dropped.

Reply:
I've added `default-fast.cnf` back. I will address my criticisms of it in a different PR.